pbcopy

```diff
@@ -0,0 +1 @@
+"6076193a-a17b-4eae-a253-1b630485b822"
```

Comment:
Whoops, let's delete this.

I meant to say `rails runner 'p SecureRandom.uuid' | pbcopy`, not `>`.